### PR TITLE
add milestone 3 data

### PR DESCRIPTION
**Pull request by petushka1**
# Milestone-3
## Add foreign keys and retrieve relational data

- [x] Add species table
- [x] Add owner table
- [x] set foreign keys

### Retrieve joined data from tables
- [x] What animals belong to Melody Pond?
image

- [x] List of all animals that are pokemon (their type is Pokemon).
image

- [x] List all owners and their animals, remember to include those that don't own any animal.
image

- [x] How many animals are there per species?
image

- [x] List all Digimon owned by Jennifer Orwell.
image

- [x] List all animals owned by Dean Winchester that haven't tried to escape.
image

- [x] Who owns the most animals?
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -15,3 +15,97 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, true, 22);
+
+/* Milestone-3 */
+/* Remove column species from table animals */
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD COLUMN owner_id INT;
```
Comment (tresorsawasawa) [inline data.sql]: - [ ] Kindly make sure you move the above queries in the `schema.sql` file as required for the project.

File: data.sql [hunk 2]
```diff
@@ -15,3 +15,97 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+        FROM (
+            SELECT COUNT(s.id) total_animals
+            FROM owner s
+            JOIN animals a1 ON a1.owner_id = s.id
+            GROUP BY s.id
+        ) AS tt
+    );
+
+
+
```
Comment (tresorsawasawa) [inline data.sql]: - [ ] Kindly make sure you move all the selected code lines in the `queries.sql` file as required for the project.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @petushka1 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @petushka1, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**